dali/imgcodec/util/convert.cc

```diff
@@ -24,24 +24,23 @@
 namespace dali {
 namespace imgcodec {
 
-void Convert(SampleView<CPUBackend> out, TensorLayout out_layout, DALIImageType out_format,
+void DLL_PUBLIC Convert(
```

Comment:
AFAIK this should be on a declaration, not definition.

Reply:
Done